qiskit_aqua/svm_classical/rbf_svc_estimator.py

```diff
@@ -15,12 +15,13 @@
 # limitations under the License.
 # =============================================================================
 
-from qiskit_aqua.multiclass.estimator import Estimator
-from sklearn.svm import LinearSVC
+from qiskit_aqua.utils.multiclass.estimator import Estimator
```

Comment:
Please reorder the import statement.

The order should be categorized into three sections, and one newline between each section.
1. standard library
2. third-party library
3. local library

Reply:
done